docs/admin-panel/hierarchy/ERC721/contract.md

```diff
@@ -15,7 +15,13 @@ ERC721 contracts are being deployed using [Contract Manager](/docs/admin-panel/m
 This information is passed directly to blockchain
 
 - **template** - Smart contract template with its unique functions
-    - **SIMPLE** - Simple template includes all basic functions like _mint_, _burn_ and _transfer_
+    - **SIMPLE** - Simple template includes all basic functions like ***mint***, ***burn*** and ***transfer***. Consist of several contracts:
+      - **Access Control** - provides the ability to restrict or grant permission for specific addresses to interact with a smart contract. (see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/access-control))
+      - **Burnable** -  allows for a token to be destroyed, reducing the overall supply and potentially increasing the value of remaining tokens (see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Burnable))
+      - **Enumerable** - allows the contract to be queried for a list of all token holders and their token balances (see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Enumerable))
+      - **Royalty** - allows for a portion of the sale price to be paid to the original creator of the NFT each time it is resold
+      - **BaseUrl** - allows the NFTs to have a URI pointing to a public location where additional information and resources can be stored
+      - **MetaData Getter** - <u>...</u>
     - **GRADED** - Extends **SIMPLE** plus _metadata storage_, which can be used for updating level of the item
```

Comment:
nope, metadata is already in simple to store templateId
Graded just has functions to update it

Reply:
Old code, right now don't have it